### PR TITLE
feat: support instance type arm for clients

### PR DESCRIPTION
**Pull request by denise89p**

### CODE REVIEW

File: modules/clients/variables.tf [hunk 1]
```diff
@@ -116,3 +116,7 @@ variable "use_autoscaling_group" {
   description = "Use autoscaling group"
   default     = false
 }
+
+variable "client_arch" {
```
Comment (assafgi) [inline modules/clients/variables.tf:120]: should be just `arch` since we are already in the context of client

File: variables.tf [hunk 2]
```diff
@@ -464,6 +464,16 @@ variable "clients_custom_data" {
   default     = ""
 }
 
+variable "client_arch" {
+  type        = string
+  default     = "x86_64"
+  description = "Type of client ami id, value can be arm/x86"
+  validation {
+    condition     = var.client_arch == "arm64" || var.client_arch == "x86_64"
```
Comment (assafgi) [inline variables.tf:472]: Nice! important

File: modules/clients/variables.tf (hunk 1 above)
Comment (jacky-mahadab) [reply modules/clients/variables.tf:120]: @assafgi - we soon be supporting arm on backends as well

Comment (assafgi) [reply modules/clients/variables.tf:120]: @jacky-mahadab you are inside client module here

Comment (assafgi) [reply modules/clients/variables.tf:120]: on the root module we will still have `client_arch`

Comment (jacky-mahadab) [reply modules/clients/variables.tf:120]: ok

Comment (denise89p) [reply modules/clients/variables.tf]: fixed

**Pull request merged**